x-pack/plugins/lens/public/editor_frame_service/editor_frame/workspace_panel/workspace_panel_wrapper.scss

```diff
@@ -34,6 +33,8 @@
   // Color the whole panel instead
   background-color: transparent !important; // sass-lint:disable-line no-important
   border: none !important; // sass-lint:disable-line no-important
+  width: 100%;
+  height: 100%;
```

Comment:
I don't see why this behavior changed, can you explain?

Reply:
Yes, so we have this style:
```
.lnsWorkspacePanelWrapper .lnsWorkspacePanelWrapper__pageContentBody > * {
    flex: 1 1 100%;
    display: flex;
    align-items: center;
    justify-content: center;
    overflow: hidden;
```

that before was applied directly to .lnsWorkspace. Now, when I added  `.lnsDragDrop__container` as a parent of the element, `.lnsDragDrop__container` got the styles above so I needed to ensure the child (.lnsWorkspace) was spread inside.